src/NuGet.Clients/NuGet.PackageManagement.UI/PackageItemLoader.cs

```diff
@@ -276,6 +276,14 @@ public IEnumerable<PackageItemViewModel> GetCurrent()
                     }
                 }
 
+                var packageLevel = metadata.TransitiveOrigins != null ? PackageLevel.Transitive : PackageLevel.TopLevel;
+
+                var transitiveToolTipMessage = string.Empty;
+                if (packageLevel == PackageLevel.Transitive)
+                {
+                    transitiveToolTipMessage = string.Format(Resources.PackageVersionWithTransitiveOrigins, metadata.Identity.Version, string.Join(", ", metadata.TransitiveOrigins), Thread.CurrentThread.CurrentCulture);
```

Comment:
Sorry, but, I just realize correct culture is CurrentUICulture

```suggestion
                    transitiveToolTipMessage = string.Format(Resources.PackageVersionWithTransitiveOrigins, metadata.Identity.Version, string.Join(", ", metadata.TransitiveOrigins), CultureInfo.CurrentUICulture);
```